libraries/matrix/impl/build.gradle.kts

```diff
@@ -29,8 +29,12 @@ anvil {
 }
 
 dependencies {
-    // implementation(projects.libraries.rustsdk)
-    implementation(libs.matrix.sdk)
+    if (file("${rootDir.path}/libraries/rustsdk/matrix-rust-sdk.aar").exists()) {
```

Comment:
Just for safety, could we make it so that this `if` works only for debug builds? (i.e. release builds will always use `   implementation(libs.matrix.sdk)` no matter what?)

Reply:
Yeah but adding a `releaseImplementation()` and haveing the if using only `debugImplementation` could be an additional layer of safety

Reply:
You mean having something like:

```kotlin
    releaseImplementation(libs.matrix.sdk)
    if (file("${rootDir.path}/libraries/rustsdk/matrix-rust-sdk.aar").exists()) {
        println("\nNote: Using local binary of the Rust SDK.\n")
        debugImplementation(projects.libraries.rustsdk)
    } else {
        debugImplementation(libs.matrix.sdk)
    }
```
?

I am wondering now if we may want to test the release build with a local SDK :)

Reply:
Yes. That's what I had in mind. In the rare case we'd like to test a release build with a local sdk  we'll have to manually change the gradle file.
This seems to me an added level of safety. But if you feel I'm being paranoid don't be afraid to say it, sometime I am.

Reply:
Works for me. Better safe than sorry!